exporter.go

```diff
@@ -182,6 +189,29 @@ func main() {
 	kingpin.Parse()
 
 	// Prepare perflib queryBuf
+	var queryBuf bytes.Buffer
+
+	// Get all existing objects if one of the perflib.objects.names flags was used
+	var objects []*perflib.PerfObject
+	if len(*perfObjectsNames) > 0 || len(*perfObjectsNamesAdd) > 0 || len(*perfObjectsNamesRemove) > 0 {
+		o, err := perflib.QueryPerformanceData("Global")
+		if err != nil {
+			panic(err)
+		}
+		objects = o
+	}
+
+	if perfObjectsNames != nil && len(*perfObjectsNames) > 0 {
+		loopPerfObjectsNames:
+		for _, p := range *perfObjectsNames {
```

Comment:
This repeated inner loop is a good candidate for refactoring into a helper method

Reply:
Done